layouts/_default/baseof.html

```diff
@@ -11,6 +11,7 @@
             {{ block "main" . }}
             {{ end }}
         </main>
+        <pulumi-top-button></pulumi-top-button>
```

Comment:
I _think_ we only want this to appear in the docs, so this should probably be placed in one those layouts. You might put it in one of the partials; [right-nav](https://github.com/pulumi/docs/blob/master/layouts/partials/docs/right-nav.html) might be an okay place for it.